src/apps/btc/btc_sign.c

```diff
@@ -305,9 +307,17 @@ app_btc_sign_error_t app_btc_sign_input(
     if (_state != STATE_INPUTS_PASS1 && _state != STATE_INPUTS_PASS2) {
         return _error(APP_BTC_SIGN_ERR_STATE);
     }
-    if (request->sequence != 0xffffffff) {
+    if (request->sequence == 0xffffffff && _locktime > 0) {
```

Comment:
this is wrong, locktime can be >0. if any other input has sequence < MAX INT it is compared to the blockheight or time, otherwise it can be anything but will be ignored.

Reply:
I'll remove the check and move the sequence check to `_locktime_applies`.